source/includes/_rest-api.md

```diff
@@ -202,6 +218,28 @@ request(options, function (error, response, body) {
 
 ```
 
+```matlab
```

Comment:
The other code examples here all return a JSON object, however this is a much more robust example (from what I can tell from the code, I don't have matlab). I'd recommend maintaining the same style of request as the other languages. If there's a more robust example, then it can be referenced separately (as you do in the Matlab Examples section).

So basically, my thinking is that the example should end at line 228, with an output of the rest data that directly correlates to the JSON displayed below the snippet (e.g. "The above command returns JSON structured like this:").